core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -5430,6 +5430,43 @@ private void checkLiteral2(String expression, String expected) {
         .ok(expected);
   }
 
+
+  @Test void testTINYINTRedshift() {
+    String query = "SELECT CAST(\"department_id\" AS TINYINT) FROM \"employee\"";
+    String expected = "SELECT CAST(\"department_id\" AS \"int2\")\n"
+        + "FROM \"foodmart\".\"employee\"";
+    sql(query)
+        .withRedshift()
+        .ok(expected);
+  }
+
+  @Test void testtinyintRedshift() {
```

Comment:
To keep things uniform with existing tests in the class rename to:
`testRedshiftCastToTinyint` (see `testMysqlCastToBigint` for instance). You could also name it `testCastToTinyint`, leaving it a bit more general (similar to `testCastToVarchar`), which could be extended later for different dialects.

Reply:
Done !